pkg/analyze/host_tls.go

```diff
@@ -0,0 +1,86 @@
+package analyzer
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/troubleshoot/pkg/analyze/types"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+)
+
+type AnalyzeHostTLS struct {
+	hostAnalyzer *troubleshootv1beta2.TLSAnalyze
+}
+
+func (a *AnalyzeHostTLS) Title() string {
+	return hostAnalyzerTitleOrDefault(a.hostAnalyzer.AnalyzeMeta, "TLS")
+}
+
+func (a *AnalyzeHostTLS) IsExcluded() (bool, error) {
+	return isExcluded(a.hostAnalyzer.Exclude)
+}
+
+func (a *AnalyzeHostTLS) Analyze(
+	getCollectedFileContents func(string) ([]byte, error), findFiles getChildCollectedFileContents,
+) ([]*AnalyzeResult, error) {
+
+	collectorName := a.hostAnalyzer.CollectorName
+	if collectorName == "" {
+		collectorName = "result"
```

Comment:
does this mean you can only have one tls collector?

Reply:
unless you give it a name yeah, your suggestion is a good one